src/main/kotlin/app/views/TaskCLI.kt

```diff
@@ -1,12 +1,15 @@
 package app.views
-
 import app.services.TaskService
-
+import java.util.Scanner
 class TaskCLI(private val taskService: TaskService) {
+    private val scanner = Scanner(System.`in`)
     fun start() {
         while (true) {
             printMenu()
-            when (readLine()?.trim()) {
+//            val choice = readLine()?.trim()
+            val choice = scanner.nextLine().trim()
+            println("Debug: User input: '$choice'" )
+            when (choice) {
                 "1" -> createTask()
```

Comment:
Might I ask what the purpose of these changes are for too? It isn't too aparrent for me.